### DISCUSSION TITLE
Discussion topics for Jun 27, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#5934] [Show and tell]
1. @carlosfelix2 Announce HL7 to FHIR translation informational meeting to share knowledge on how the translation is done.

**Comment by PatriciaPerozoUSDS** [comment]
@PatriciaPerozoUSDS Updating [class diagram](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/docs/engineering.md) from 2020 to make onboarding easier.

**Comment by bgantick** [comment]
Practices around rebase vs merge (related: terraform issues failing builds)
